package/linux-firmware/linux-firmware.mk

```
@@ -542,6 +542,8 @@ LINUX_FIRMWARE_AX210_UCODE_API_MAX = 83
 # are seemingly tracking the 22000 series.
 LINUX_FIRMWARE_AX210_UCODE_API_MAX_9560 = 77
 
+LINUX_FIRMWARE_GL_UCODE_API_MAX = 83
```

I'd like to keep this aligned with kernel naming to make updates a little easier. Also, keep it in one block:
```suggestion
LINUX_FIRMWARE_IWL_BZ_UCODE_API_MAX = 83
```

---

This one is incomplete, I'll commit it right way